ocp4/profiles/coreos-ncp.profile

```diff
@@ -227,49 +220,51 @@ selections:
     #- package_abrt_removed
 
     ### Login
-    #- disable_users_coredumps
-    #- sysctl_kernel_core_pattern
-    #- coredump_disable_storage
-    #- coredump_disable_backtraces
-    #- service_systemd-coredump_disabled
+    - disable_users_coredumps
+    - sysctl_kernel_core_pattern
+    - coredump_disable_storage
+    - coredump_disable_backtraces
+    - service_systemd-coredump_disabled
     #- var_accounts_max_concurrent_login_sessions=10
     #- accounts_max_concurrent_login_sessions
     #- securetty_root_login_console_only
     #- var_password_pam_unix_remember=5
     #- accounts_password_pam_unix_remember
 
     ### SELinux Configuration
-    #- var_selinux_policy_name=targeted
-    #- selinux_policytype
+    - var_selinux_state=enforcing
+    - selinux_state
+    - var_selinux_policy_name=targeted
+    - selinux_policytype
 
     ### Application Whitelisting (RHEL 8)
-    #- package_fapolicyd_installed
-    #- service_fapolicyd_enabled
-    #- configure_fapolicyd_mounts
+    - package_fapolicyd_installed
+    - service_fapolicyd_enabled
+    - configure_fapolicyd_mounts
```

Comment:
Very interesting. I wonder if a) anyone has tested fapolicyd with OCP and b) if fapolicyd should run on the host or in a deamonset pods. Something to think about and test I guess..

Reply:
If this is required by a control, we gotta bring it up to the coreOS team and request the daemon.

Reply:
This is required functionality... It will also have to be expanded to containers and the openshift env.... but that is for another time.

Reply:
@redhatrises could you create a Jira for us to start working with the CoreOS team to include this in the distribution?

Reply:
@JAORMX yes. Can do. This is part of the combined team assessing CoreOS/OCP.